src/topic.ts

```diff
@@ -460,16 +519,15 @@ export class Topic {
           reqOpts,
           gaxOpts,
         },
-        // tslint:disable-next-line only-arrow-functions
-        function() {
-          const subscriptions = arguments[1];
+        (...args) => {
```

Comment:
That's still going to need types, right?

Reply:
(assuming `: any[]`)

Reply:
yes its any[], do i need to explicitly declare that, i am not getting error after enabling noImplicityAny for this

Reply:
can [`ListTopicSubscriptionsResponse`](https://github.com/googleapis/nodejs-pubsub/pull/420/files#diff-b0559621e6dc13b8ee64042d79d2f75fR1546) be used?